src/plotting.jl

```diff
@@ -300,9 +300,15 @@ end
 
 """
     evaluation_metrics_plot(data::Dict; resolution=(1000, 1000), textsize=12)
+    evaluation_metrics_plot(row::EvaluationRow; kwargs...)
 
-Plots all evaluation metrics Lighthouse has to offer.
+Plot all evaluation metrics generated via [`evaluation_metrics_row`](@ref) and/or
+[`evaluation_metrics`](@ref) in a single image.
 """
+function evaluation_metrics_plot(row::EvaluationRow; kwargs...)
+    return evaluation_metrics_plot(_observation_table_to_inputs(row); kwargs...)
```

Comment:
does this work? I thought it would need to be something like `_evaluation_table_to_inputs`

Reply:
:headdesk: yup, 100%. fixed.